### PR TITLE
*: cut 0.3.0 release

### PR DESCRIPTION
**Pull request by asymmetric**
0.3.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Habitat operator CHANGELOG
 
+## [v0.3.0](https://github.com/kinvolk/habitat-operator/tree/v0.2.0) (2017-20-11)
```
Comment (asymmetric) [inline CHANGELOG.md]: Unrelated to this PR, but do we want to instead order by DD-MM-YYYY? That's less [US centric](https://en.wikipedia.org/wiki/Date_format_by_country).

Let me know what you think and I'll open a PR if we need to.

Comment (asymmetric) [reply CHANGELOG.md]: And [here](https://github.com/kinvolk/habitat-operator/pull/141)'s the PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+
+### Features & Enhancements
+
+- Start multiple parallel queue workers [#136](https://github.com/kinvolk/habitat-operator/pull/136)
+- Wait for caches to be synced before starting workers [#134](https://github.com/kinvolk/habitat-operator/pull/134)
+- Explicitly stop workers when the controller stops [#135](https://github.com/kinvolk/habitat-operator/pull/135)
+
+### Bug fixes
+
+- Prevent panics due to enqueueing `nil` objects [#133](https://github.com/kinvolk/habitat-operator/pull/133)
```
Comment (LiliC) [inline CHANGELOG.md]: `Prevent panic due to ...`

File: CHANGELOG.md (hunk 1 above)
Comment (LiliC) [reply CHANGELOG.md]: Approved the PR, don't have a strong preference.  Just don't forget to change it here now. :)

**Pull request merged**